test.js

```diff
@@ -134,6 +134,52 @@ test('execa.shellSync() includes stdout and stderr in errors for improved debugg
 	t.throws(() => execa.shellSync('node fixtures/error-message.js'), {message: STDERR_STDOUT_REGEXP, code: 1});
 });
 
+test('execa() with .kill after it with SIGKILL should kill cleanly', async t => {
+	const subprocess = execa('node', ['fixtures/sub-process-no-killable']);
+
+	subprocess.kill('SIGKILL');
+
+	try {
+		await subprocess;
+		t.fail('Didnt expect success');
+	} catch (error) {
+		t.deepEqual(error.signal, 'SIGKILL');
```

Comment:
```suggestion
		t.is(error.signal, 'SIGKILL');
```

Does a shallow comparison instead of deep (faster).

Reply:
Fixed: 6e9424db6017cc6268ae0201aacca520c2074c53